integration/mesos/src/main/java/alluxio/mesos/AlluxioFramework.java

```diff
@@ -61,9 +61,11 @@ public void run() {
     if (Configuration.containsKey(PropertyKey.INTEGRATION_MESOS_ROLE)) {
       frameworkInfo.setRole(Configuration.get(PropertyKey.INTEGRATION_MESOS_ROLE));
     }
-    // Setting the user to an empty string will prompt Mesos to set it to the current user.
     if (Configuration.containsKey(PropertyKey.INTEGRATION_MESOS_USER)) {
       frameworkInfo.setUser(Configuration.get(PropertyKey.INTEGRATION_MESOS_USER));
+    } else {
+      // Setting the user to an empty string will prompt Mesos to set it to the current user.
+      frameworkInfo.setUser("");
```

Comment:
Should we just set this as the default value?

Reply:
It's confusing to treat empty string different from unset/null. I made it this way so that we treat empty string and unset identically. The original bug probably occurred because someone assumed empty string would be treated the same as not setting it.